client/packages/host/src/components/Sync/Sync.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { PropsWithChildren, useState, useEffect } from 'react';
+
+import {
+  Grid,
+  LoadingButton,
+  RadioIcon,
+  Typography,
+  useFormatDateTime,
+  useTranslation,
+} from '@openmsupply-client/common';
+import { useHost } from '../../api/hooks';
+import { SyncProgress } from '../SyncProgress';
+
+const STATUS_POLLING_INTERVAL = 1000;
+
+const useSync = () => {
+  // Polling whenever Sync page is opened
+  const { syncStatus, numberOfRecordsInPushQueue } = useHost.utils.syncInfo(
+    STATUS_POLLING_INTERVAL
+  );
+  const { mutateAsync: manualSync } = useHost.sync.manualSync();
+
+  // true by default to wait for first syncStatus api result
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    if (!syncStatus) {
+      return;
+    }
+    // When we receive syncStatus, resulting isLoading state should be = isSyncing form api result
+    setIsLoading(false);
+  }, [syncStatus]);
+
+  const onManualSync = async () => {
+    // isLoading is reset on next result of polled api query
+    setIsLoading(true);
+    await manualSync();
+  };
+
+  return {
+    isLoading: !!syncStatus?.isSyncing || isLoading,
+    latestSyncDate: syncStatus && new Date(syncStatus.summary.started),
```

Comment:
I've added to: https://github.com/openmsupply/open-msupply/issues/688
Was tempted to do it here, but I think it's better as a tidy up issue (get this beast merged).

Also Josh pointed out that hooks [shouldn't have presentation ](https://github.com/openmsupply/open-msupply/pull/685#discussion_r985451350), so I ended up just returning parsed Date rather then formatted string of date. 

I am not UI guru but I think it would be hard to translate `which is`, I would have thought that localisedDistanceToNow + ago would be sufficient.

> Side question: the started and finished aren't changing when I sync, and they say 13h ago? 

They should : / 
![last sync updates](https://user-images.githubusercontent.com/26194949/193926049-bec07c2a-8fc8-4cf4-b4fd-dbcb2812cea4.gif)